qiskit_ibm_provider/job/ibm_circuit_job.py

```diff
@@ -831,10 +856,12 @@ def _set_result(self, raw_data: Optional[Dict]) -> None:
         if raw_data is None:
             self._result = None
             return
-        raw_data["client_version"] = self.client_version
-        decode_result(raw_data)
+        self._check_result_for_errors(raw_data)
+        # TODO: check whether client version can be extracted from runtime data
+        # raw_data["client_version"] = self.client_version
```

Comment:
IIRC client version is returned by the `metrics` endpoint.

Reply:
Done